spec/lib/cooked_post_processor_spec.rb

```diff
@@ -1456,6 +1456,28 @@
           HTML
         end
 
+        context "with video upload" do
+          SiteSetting.authorized_extensions = "png|jpg|gif|mp4"
```

Comment:
Should this be in a before block? Also you probably just need: `SiteSetting.authorized_extensions = "mp4"`